x-pack/plugin/sql/jdbc/src/main/java/org/elasticsearch/xpack/sql/jdbc/jdbc/JdbcPreparedStatement.java

```diff
@@ -378,4 +514,12 @@ public boolean execute(String sql, String[] columnNames) throws SQLException {
     public long executeLargeUpdate() throws SQLException {
         throw new SQLFeatureNotSupportedException("Batching not supported");
     }
+    
+    private boolean setIfNull(int parameterIndex, Object o, int type) throws SQLException {
```

Comment:
I think it might be easier to read the code without this method to be honest. It saves a line every time you call it makes me go "what is going on here?" every time I see it. Not sure.